packages/graphql/src/api-v6/schema-generation/schema-types/TopLevelEntitySchemaTypes.ts

```diff
@@ -75,12 +75,22 @@ export class TopLevelEntitySchemaTypes extends EntitySchemaTypes<TopLevelEntityT
             name: this.entity.typeNames.queryField,
             type: this.connectionOperation,
             args: {
-                where: this.filterSchemaTypes.operationWhere,
+                where: this.filterSchemaTypes.operationWhereTopLevel,
             },
             resolver,
         });
     }
 
+    // protected get connectionSort(): InputTypeComposer {
+    //     return this.schemaBuilder.getOrCreateInputType(this.entityTypeNames.connectionSort, () => {
+    //         return {
+    //             fields: {
+    //                 node: this.nodeSort.NonNull.List,
+    //             },
+    //         };
+    //     });
+    // }
```

Comment:
```suggestion
```

Reply:
This will be sorted out in a follow-up PR for sorting